RationalOptionPages.php

```diff
@@ -253,7 +253,7 @@ public function admin_init() {
 						$params['callback'] = array( $this, $params['callback'] );
 					}
 			
-					call_user_func_array( 'add_settings_section', $params );
+					call_user_func_array( 'add_settings_section', array_values( $params ) );
```

Comment:
Would it be an option to do something like this, here and below?

```suggestion
					add_settings_section( ...array_values( $params ) );
```

Reply:
Yes, I think that's a good improvement.